src/index.html

```diff
@@ -6,7 +6,37 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="calendar calendar--start-day--mon calendar--month-length--31">
+    <div class="calendar__cell calendar__day--1"></div>
```

Comment:
```suggestion
    <div class="calendar__cell"></div>
```
Remove it everywhere. Use nth-child instead